Extension/src/LanguageServer/client.ts

```diff
@@ -1026,6 +1026,8 @@ export class DefaultClient implements Client {
                 return;
             }
             settings.defaultCompiler = result[0].fsPath;
+            action = "compiler browsed";
+            telemetry.logLanguageServerEvent('compilerBrowsed', { action });
```

Comment:
I'd like to keep this in the same 'compilerSelection' event if possible.  Maybe we wrap the whole function in a `try/finally` so that we can log it whenever this function exits since there are so many early returns.